mitodl/open-discussions
# Filter out indexing user from moderator and contributor lists

#### What are the relevant tickets?
Fixes #891 

#### What's this PR do?
Remove channel creator from contributor and moderator list in UI. No changes are made to the REST API, though it won't matter since no user will have permission to remove that user as moderator, and their contributor status doesn't matter if they are a moderator.

#### How should this be manually tested?
View the contributor and moderator list. You should not see any user with `<missing>`.
